airflow-core/src/airflow/ui/src/pages/Dag/Header.tsx

```diff
@@ -108,7 +111,34 @@ export const Header = ({
               />
             )}
             <FavoriteDagButton dagId={dag.dag_id} withText={true} />
-            <ParseDag dagId={dag.dag_id} fileToken={dag.file_token} />
+            <Menu.Root>
+              <Menu.Trigger asChild>
+                <Button aria-label={translate("dag:header.buttons.advanced")} variant="outline">
+                  <LuMenu />
+                </Button>
+              </Menu.Trigger>
+              <Portal>
+                <Menu.Positioner>
+                  <Menu.Content>
+                    <Menu.Item value="reparse">
+                      <div style={{ width: "100%" }}>
```

Comment:
You don't need that. It's because of the fragment that is inside the `DeleteDagButton`.

Reply:
You can apply the width to the outer element and change the react fragment into a `Box` container and apply the width there, that should solve your issue.